taskt/Core/Automation/Commands/Loop Commands/LoopListCommand.cs

```diff
@@ -25,17 +25,17 @@ public class BeginListLoopCommand : ScriptCommand
         [Attributes.PropertyAttributes.Remarks("Use this command to iterate over the results of the Split command.")]
         public string v_LoopParameter { get; set; }
 
-        public BeginListLoopCommand()
+        public LoopListCommand()
         {
-            this.CommandName = "BeginListLoopCommand";
-            this.SelectionName = "Loop List";
+            this.CommandName = "LoopListCommand";
+            this.SelectionName = "List Loop";
```

Comment:
I think you got confused here. Should be "Loop List"

Reply:
done